### PR TITLE
[AF-1629]: Added credentials to import external project

### PR DESCRIPTION
**Pull request by adrielparedes**
@ederign this https://github.com/kiegroup/kie-wb-common/commit/b106e3faa0bcdf86bd84c04a1eeacbca9adfd785 is already merged, and it is in master, or Am I missing something?

### CODE REVIEW

File: kie-wb-common-screens/kie-wb-common-examples-screen/kie-wb-common-examples-screen-api/src/main/java/org/kie/workbench/common/screens/examples/model/Credentials.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        if (!(o instanceof Credentials)) {
+            return false;
+        }
+
+        Credentials that = (Credentials) o;
+
+        if (username != null ? !username.equals(that.username) : that.username != null) {
+            return false;
+        }
+        return !(password != null ? !password.equals(that.password) : that.password != null);
```
Comment (tomasdavidorg) [inline kie-wb-common-screens/kie-wb-common-examples-screen/kie-wb-common-examples-screen-api/src/main/java/org/kie/workbench/common/screens/examples/model/Credentials.java]: Is this automatically generated by IntelliJ? I am just wondering why there is so many not operators.

Comment (adrielparedes) [reply kie-wb-common-screens/kie-wb-common-examples-screen/kie-wb-common-examples-screen-api/src/main/java/org/kie/workbench/common/screens/examples/model/Credentials.java]: @tomasdavidorg I think got too excited with operators, I will try to simplify it. 
Also the test that is faling is not related.
Thanks for the review!

Comment (adrielparedes) [reply kie-wb-common-screens/kie-wb-common-examples-screen/kie-wb-common-examples-screen-api/src/main/java/org/kie/workbench/common/screens/examples/model/Credentials.java]: @tomasdavidorg I double checked operators and I think they are right, they cover all alternatives :)

Comment (tomasdavidorg) [reply kie-wb-common-screens/kie-wb-common-examples-screen/kie-wb-common-examples-screen-api/src/main/java/org/kie/workbench/common/screens/examples/model/Credentials.java]: @adrielparedes Thanks :slightly_smiling_face:

**Comment by ederign** [comment]
Error is not related (Failed to open server socket for /127.0.0.1:41658)

**Comment by jhrcek** [comment]
@adrielparedes @ederign shouldn't this be merged (as a blocker) into 7.11.x branch? Not sure how this will be handled at the prod. level (CC @mareknovotny)

**Pull request merged**